sys/shell/commands/sc_icmpv6_echo.c

```diff
@@ -272,6 +272,14 @@ int _icmpv6_ping(int argc, char **argv)
             puts("ping timeout");
         }
 
+        while(msg_try_receive(&msg) > 0) {
```

Comment:
Is https://github.com/RIOT-OS/RIOT/pull/4031/files#diff-86b72948b0111bd23b9542d354c67781R295 then still necessary?